### ISSUE TITLE
Threshold for `probe_avg_mag_sqrd_c` specified in different units then it reports.

### ISSUE THREAD

**Issue by fake-name** [gnuradio/gnuradio#1163]
Basically, the `analog.probe_avg_mag_sqrd_c()` probe has it's units specified in dB, but it returns it's internal value in another format (raw average of the magnitude squared of the the input). 

This gets confusing quickly, when you use the `analog.probe_avg_mag_sqrd_c().level()` interface to see what your signal levels are, set the threshold based on those units (note that the units for `analog.probe_avg_mag_sqrd_c().level()` are not specified), and then you have a signal that never unmutes.

Ideally, either the units for `analog.probe_avg_mag_sqrd_c().level()` should be converted to dB, or the units you use to specify the threshold should be in the same scale as `level()` returns.

Basically, it's possible to have a context where: `probe.level() > probe.threshold()` is `True`, but `probe.unmuted()` is `False`. I only figured out what was going on by actually looking at the C++ source.

**Comment by jmcorgan** [comment]
Closing as submitted PR was withdrawn.
